src/js/modules/infragistics.ui.scroll.js

```diff
@@ -1197,7 +1197,7 @@
 		_linkElementsH: function (inElements) {
 			this._linkedHElems = [];
 			if (inElements) {
-				for (var index in inElements) {
+				for (var index = 0; index < inElements.length; index++) {
```

Comment:
@skrustev OK, if you expect an array then a regular for-loop works for this case.